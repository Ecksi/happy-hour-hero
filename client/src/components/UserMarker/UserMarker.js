import React, { Component } from 'react';
import { connect } from 'react-redux';
import './UserMarker.css';
import userMarker from './assets/user-marker.png';

export class UserMarker extends Component {
  render() {
    return (
      <section className="userMarker">
        <img src={ userMarker } alt="User Pin"/>
      </section>
    );
  }
}

export const mapStateToProps = (state) => ({
  location: state.location
});

export default connect(mapStateToProps)(UserMarker);
