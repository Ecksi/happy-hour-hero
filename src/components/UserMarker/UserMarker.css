.fa-male {
  font-size: 4rem;
  color: #3c3a3a; }
