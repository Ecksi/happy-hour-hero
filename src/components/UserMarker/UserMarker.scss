.fa-male {
  font-size: 4rem;
    color: #3c3a3a;
}

.userMarker {
  width: 60px;
  height: 20px;
  background: purple;
  padding: 10px;
  display: flex;
  align-items: center;
  color: white;
  border-radius: 4px;
  justify-content: center; 
  text-align: center; 
}