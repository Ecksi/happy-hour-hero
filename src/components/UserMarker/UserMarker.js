import React, { Component } from 'react';
import { connect } from 'react-redux';
import './UserMarker.css';
import userMarker from './assets/user-marker.png';


class UserMarker extends Component {
  render() {
    return (
      <section className="userMarker">
        <img src={ userMarker } />
      </section>
    );
  }
}

export const mapStateToProps = (state) => ({
  location: state.location
});

export default connect(mapStateToProps)(UserMarker);
