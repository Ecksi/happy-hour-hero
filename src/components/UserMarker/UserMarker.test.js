import React from 'react';
import { shallow } from 'enzyme';
import { UserMarker } from './UserMarker';

describe('UserMarker', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<UserMarker />);
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});