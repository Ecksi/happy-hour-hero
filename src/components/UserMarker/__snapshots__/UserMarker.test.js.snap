// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserMarker matches the snapshot 1`] = `undefined`;
